python/events/evaluation_tools/events/event_detection/decomposition.py

```diff
@@ -149,7 +148,7 @@ def mark_event_flows(
         window: int, offset, or BaseIndexer subclass, required
             Size of the moving window for `pandas.Series.rolling.min`.
             This filter is used to model the trend in `series`.
-        minimum_event_duration: str, tuple, datetime.timedelta, pd.tseries.offsets.DateOffset, optional, default '0H'
+        minimum_event_duration: pandas.Timedelta, datetime.timedelta, numpy.timedelta64, str, int, optional, default 0
```

Comment:
I liked what the default before, I think it was more intuitive. But that's a minor thing, either way.

Reply:
Changed back